src/go/k8s/pkg/resources/statefulset_update.go

```diff
@@ -40,6 +40,9 @@ const (
 	// requeue resource reconciliation.
 	RequeueDuration        = time.Second * 10
 	defaultAdminAPITimeout = time.Second * 2
+
+	ManagedDecommissionAnnotation = "operator.redpanda.com/managed-decommission"
+	PodUpdateAnnotation           = "operator.redpanda.com/pending-update"
```

Comment:
NIT: This commit introduces unused const.

Reply:
Yeah, it's not there. Looks like you're reviewing an outdated commit.